### PR TITLE
Feat: new envelope + dragBounds

### PR DESCRIPTION
**Pull request by katspaugh**
* The Envelope plugin now supports multiple points and has a new API.
* I've added a new multitrack option: `dragBounds` which, if set to true, doesn't allow to drag tracks past zero.
* Fixed a bug which prevented the endCue to be dragged.
* Migrate from webpack to rollup + live-server.

Screenshot 2023-08-27 at 12 03 43

### CODE REVIEW

**Comment by katspaugh** [comment]
@PhiloNL please take a look. This will be released as 0.2.0.

File: src/multitrack.ts [hunk 1]
```diff
@@ -188,7 +190,7 @@ class MultiTrack extends EventEmitter<MultitrackEvents> {
           })
           const endCueRegion = wsRegions.addRegion({
             start: endCue,
-            end: endCue + this.durations[index],
```
Comment (katspaugh) [inline src/multitrack.ts:191]: This was a bug.

File: yarn.lock [hunk 2]
```diff
@@ -2650,10 +2650,10 @@ watchpack@^2.4.0:
     glob-to-regexp "^0.4.1"
     graceful-fs "^4.1.2"
 
-wavesurfer.js@^7.1.5:
-  version "7.1.5"
-  resolved "https://registry.yarnpkg.com/wavesurfer.js/-/wavesurfer.js-7.1.5.tgz#a38febc554fa76347075868f12d8e91b067f3100"
-  integrity sha512-2zb82LxQzeFn5/O5JhEs1uytdlkksUDALWt7TPN8ppP4jrajuigwcM2g4wx6b+SO4Frweg/r99L1hznr0lsBEg==
+wavesurfer.js@^7.2.0-beta.3:
+  version "7.2.0-beta.3"
```
Comment (katspaugh) [inline yarn.lock]: I'll change this to 7.2.0 once this version is released.

**Comment by PhiloNL** [comment]
Looks fine to me 👍

**Pull request merged**